backend/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
+name = "dlp-backend"
+version = "0.1.0"
+description = "Main Backend for Deep Learning Playground's Training Services"
+authors = ["DSGT-DLP <dlp@datasciencegt.org>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "dlp_backend"}]
+
+# Try not directly edit this file! Use `poetry add` and `poetry remove` instead.
+
+[tool.poetry.dependencies]
+python = ">=3.9, <3.10"
+flask-socketio = "*"
+numpy = "*"
+pandas = "*"
+matplotlib = "*"
+seaborn = "*"
+mypy = "*"
+pytest = "*"
+scikit-learn = "*"
+scipy = "1.7.3"
+pyyaml = "*"
```

Comment:
not sure if this library is used anywhere in our codebase? @dwu359 do you have context?

Reply:
pytest seems like it's used for testing python code, scikit-learn and spicy are probably used for training, I'm not entirely sure what pyyaml is at first glance

Reply:
Yeah. I think pytest is mainly for unit tests in python. I think it's mainly supposed to be a dev dependencies

Reply:
im going to resolve this for now in order to merge the pr, but let's address cleaning up dependencies in pyproject.toml in following pr